ui/imports/shared/controls/chat/PaymentRequestMiniCardDelegate.qml

```diff
@@ -25,81 +25,93 @@ CalloutCard {
-
-                StatusBaseText {
-                    Layout.fillWidth: true
-                    Layout.fillHeight: true
-                    text: qsTr("Payment request")
-                    font.pixelSize: Theme.additionalTextSize
-                    font.weight: Font.Medium
-                }
-                RowLayout {
-                    Layout.fillWidth: true
-                    Layout.fillHeight: true
-                    StatusBaseText {
-                        Layout.maximumWidth: parent.width * 0.8
-                        Layout.fillHeight: true
-                        font.pixelSize: Theme.tertiaryTextFontSize
-                        color: Theme.palette.baseColor1
-                        verticalAlignment: Text.AlignVCenter
-                        elide: Text.ElideRight
-                        text: root.amount
-                    }
-                    StatusBaseText {
-                        Layout.fillHeight: true
-                        font.pixelSize: Theme.tertiaryTextFontSize
-                        color: Theme.palette.baseColor1
-                        verticalAlignment: Text.AlignVCenter
-                        text: root.symbol
-                    }
-                }
+            StatusSmartIdenticon {
+                anchors.bottom: parent.bottom
+                anchors.horizontalCenter: parent.right
+                asset.width: 16
+                asset.height: 16
+                asset.bgColor: root.containsMouse ? Theme.palette.transparent : Theme.palette.background
+                asset.bgHeight: 20
+                asset.bgWidth: 20
+                asset.isImage: true
+                asset.name: Constants.tokenIcon(root.symbol)
             }
+        }
+
+        Item {
+            Layout.fillHeight: true
+            Layout.preferredWidth: Theme.halfPadding / 2
+            Layout.rowSpan: 3
+        }
+
+        Item {
+            // NOTE this item is added because for some reason the "Payment request" text is not rendered until hover
+            Layout.preferredHeight: 1
+            Layout.fillWidth: true
+        }
+
+        StatusFlatButton {
+            id: closeButton
+            Layout.rowSpan: 3
+            icon.name: "close"
+            size: StatusBaseButton.Size.Small
+            hoverColor: Theme.palette.directColor8
+            textColor: Theme.palette.directColor1
+            onClicked: root.close()
+        }
 
-            StatusFlatButton {
-                id: closeButton
-                icon.name: "close"
-                size: StatusBaseButton.Size.Small
-                hoverColor: Theme.palette.directColor8
-                textColor: Theme.palette.directColor1
-                onClicked: root.close()
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: qsTr("Payment request")
+            font.pixelSize: Theme.additionalTextSize
+            font.weight: Font.Medium
+        }
+
+        Item {
```

Comment:
Why 2 texts here inside another container, instead of just 1? They have the same size, color, etc...